engine/execution/computation/query/executor.go

```diff
@@ -351,6 +355,10 @@ func (e *QueryExecutor) GetAccountAvailableBalance(
 	return accountAvailableBalance, nil
 }
 
+// GetAccountKeys returns account public keys by the provided address.
+//
+// Expected error returns during normal operation:
+//   - [fvmerrors.ErrCodeAccountPublicKeyNotFoundError]: If public keys not found for the given address.
```

Comment:
I don't think this is the only possible error code. I think there could also be errors for context canceled, etc. Maybe just document `CodedError` here and let the caller figure out what they want to look for.